customTests/azure_ib_write_bw_non_gdr.nhc

```diff
@@ -8,7 +8,7 @@ SLEEP_TIME=5
 
 HOSTNAME=`hostname`
 
-function check_ib_bw(){
+function check_ib_bw_non_gdr(){
```

Comment:
could we combine functions and just pass an argument to turn on or off gdr?

Reply:
I rather leave the custom tests as simple as possible to avoid confusion/complexity.